code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -341,7 +341,7 @@
 
 /mob/living/simple_animal/bot/medbot/proc/assess_patient(mob/living/carbon/C)
 	//Time to see if they need medical help!
-	if(C.stat == DEAD || (C.status_flags & FAKEDEATH))
+	if(C.stat == DEAD || (C.status_flags & FAKEDEATH) || (ROBOTIC_LIMBS in C.dna.species.species_traits))

```

Comment:
No bueno at all. Make it iterate through a patient's bodyparts and act when at least one of them isn't robotic.

Reply:
Do you have an example of code I can look at that does this?